### ISSUE TITLE
Add title to ressource edit

### ISSUE THREAD

**Issue by bernhardh** [arsenaltech/nova-tab#17]
The default title of a ressource edit view is "Edit RESSOURCE_NAME".  So if you would edit an artikel you see "Edit Article", but you don't know, which article you are currently editing. Normally you would have a field like `Text::make("name")` or `Text::make("title")` to identify your article. But if you are using tabs, you see this "name"-field only in one tab. Let me illustrate it by an example:

I have 5 tabs. If you are on the first one, you see the "Name" Field and know which article you are currently editing:

![image](https://user-images.githubusercontent.com/642292/47098663-ed3a8300-d233-11e8-9310-5d36573597f7.png)

But if you switch to another tab, you can't see the name of the article anymore:

![image](https://user-images.githubusercontent.com/642292/47098708-06433400-d234-11e8-988c-b28df01d9642.png)

Is there maybe a way, to add the name of the ressource somewhere, where it is visible every time, somehow like so:

![image](https://user-images.githubusercontent.com/642292/47098846-47d3df00-d234-11e8-8a72-a56e53fb16c0.png)

**Comment by shabbirbharmal** [comment]
This should be possible by overriding label method in your resource class.

**Comment by bernhardh** [comment]
I don't think this is possible - how should that work? The `public static function label` is a static method.  So you don't have access to the model data itself, so no access to the model name/title field.
